Gruntfile.js

```diff
@@ -77,8 +94,8 @@ module.exports = (grunt) => {
       },
     },
     eslint: {
-      target: SOURCES.concat(['./tests/**/*.ts', './tests/**/*.js']),
-      options: { fix: true },
+      target: SOURCES.concat('./tests'),
```

Comment:
What do you mean does not match correctly? It did for me. Do you have any examples?

Reply:
Look in the code diff wih maste. You have writen the options in a complete different way and they do not match. It is not easy to see the changes. It seems that you had not used the version in master.

Reply:
I rebased. This one I changed on purpose. So what is your concern here?

Reply:
I made the changed because it was not matching the files I had the discussion in the past about the regex with you and @0xfe in the end it was not a regex but a list.

Reply:
I just checked. And it worked, it matched ts and js files.
<img width="449" alt="Screenshot 2021-04-02 at 20 39 04" src="https://user-images.githubusercontent.com/227707/113444160-9e790780-93f3-11eb-9f75-c4d2da1023f3.png">